### DISCUSSION TITLE
Welcome to asn1scc Discussions!

### DISCUSSION THREAD

**Discussion by ttsiodras** [esa/asn1scc#241] [Announcements]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ht099** [comment]
Hello, 

In the latest version 4.4.0.1 the option -BER is missing. Was it removed ? Are BER rules supported now ?

**Comment by maxime-esa** [comment]
No, `BER` is not supported by asn1scc. However it could possibly be "emulated" using ACN. I wrote an example to illustrate how to do it here: https://taste.tuxfamily.org/wiki/index.php?title=Technical_topic:_Hints_to_model_complex_packet_encodings_with_ASN.1_and_ACN#Example_2:_TLV_Encoding

And to check all the ACN possibilities, the complete user manual is here:
https://taste.tuxfamily.org/wiki/index.php?title=Technical_topic:_ASN.1_-_An_introduction_to_ACN
